octoprint_authldap/templates/authldap_settings.jinja2

```diff
@@ -0,0 +1,79 @@
+<form class="form-horizontal">
```

Comment:
Changed filename to to renaming of the plugin, also implemented settings as proposed in the best practice